### ISSUE TITLE
Access to Liked videos playlist?

### ISSUE THREAD

**Issue by orschiro** [mashlol/PlayTube#3]
Dear @mashlol,

I have another question. 

Is it possible to access my Liked Videos playlist through the PlayTube interface?

That would be cool. :)

Warmly,

Robert

**Comment by mashlol** [comment]
Assuming you mean accessing the videos you've previously liked through YouTube's liking feature, no it's not currently possible.  Importing playlists is something I'll probably add eventually.

**Comment by orschiro** [comment]
> Assuming you mean accessing the videos you've previously liked through YouTube's liking feature, no it's 

Yes, that's indeed what I was meaning. Thanks for taking this feature into consideration!

~ Robert
